tests/yam/agama/playwright_test.pm

```diff
@@ -0,0 +1,32 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run interactive installation with Agama,
+# run playwright tests directly from the Live ISO.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+
+use testapi qw(
+  assert_screen
+  assert_script_run
+  get_var
+  select_console
+  upload_logs
+);
+
+sub run {
+    my $testcase = 'take_screenshots';
+
+    assert_screen('agama_product_selection', 120);
+    $testapi::password = 'linux';
+    select_console 'root-console';
+    assert_script_run("RUN_INSTALLATION=1 playwright test --trace on --project chromium --config /usr/share/agama-playwright $testcase", timeout => 600);
+    #upload trace after test run
+    assert_script_run("playwright show-trace trace.zip");
```

Comment:
probably not much utility to show the trace to see in the screenshot that openQA does... we just need to upload it.